packages/devtools_app/lib/src/screens/memory/memory_controller.dart

```diff
@@ -68,8 +69,9 @@ class MemoryController extends DisposableController
   /// The controller is late to enable test injection.
   late final DiffPaneController diffPaneController;
 
-  /// Controller for [AllocationProfileTableView].
-  final allocationProfileController = ProfilePaneController();
+  final profilePaneController = ProfilePaneController();
+
+  final tracingPaneController = TracingPaneController();
```

Comment:
now that we are storing this in the controller, does this need to be cleared or disposed at any point by the memory controller?

Reply:
will follow up in separate PR

Reply:
https://github.com/flutter/devtools/pull/5071